src/enums/filingTypes.ts

```diff
@@ -1,6 +0,0 @@
-export enum FilingTypes {
-  INCORPORATION_APPLICATION = 'incorporationApplication',
-  REGISTRATION = 'registration',
-  RESTORATION = 'restoration',
-  VOLUNTARY_DISSOLUTION = 'dissolution',
```

Comment:
The is the source of the dissolution evil -- other UIs had a value of `voluntaryDissolution` here, which is invalid as there is no such filing type (it's called just `dissolution` and has a subtype).